pkg/api/grpcServer.go

```diff
@@ -14,314 +14,204 @@
 
 package api
 
-// gRPC server. The GRPCServer type handles incoming streaming gRPC traffic and invokes method-specific
-// handlers to implement the method-specific logic.
-//
-// When you create a GRPCServer instance, you specify a number of transport-level options, along with the
-// set of method handlers responsible for the logic of individual API methods
-
-// TODO: Once the gRPC code is updated to use context objects from "context" as
-// opposed to from "golang.org/x/net/context", this code should be updated to
-// pass the context from the gRPC streams to downstream calls as opposed to merely
-// using context.Background.
-
 import (
 	"context"
-	"io"
-	"sync"
+	"fmt"
 	"time"
 
 	"github.com/golang/glog"
-	"github.com/golang/protobuf/proto"
 	rpc "github.com/googleapis/googleapis/google/rpc"
-	"github.com/opentracing/opentracing-go/log"
+	context2 "golang.org/x/net/context"
```

Comment:
I don't know what to do to fix things. gRPC would need to make a breaking change to update to the new context definition. Otherwise, we have to pinch our nose and accept this annoyance.